db/seeds.rb

```diff
@@ -21,3 +21,9 @@
                password_confirmation: password,
                activated: true, activated_at: Time.zone.now)
 end
+
+users = User.order(:created_at).take(6)
+50.times do
```

Comment:
```suggestion
5.times do
```
mỗi người 5 bài được rồi em